src/openms/source/FORMAT/MascotRemoteQuery.cpp

```diff
@@ -354,12 +356,12 @@ namespace OpenMS
     }
 
     QByteArray querybytes;
-    querybytes.append("--" + boundary + "--\n");
+    querybytes.append(("--" + boundary + "--\n").toUtf8());
     querybytes.append("Content-Disposition: ");
     querybytes.append("form-data; name=\"QUE\"\n");
     querybytes.append("\n");
     querybytes.append(query_spectra_.c_str());
-    querybytes.append("--" + boundary + "--\n");
+    querybytes.append(("--" + boundary + "--\n").toUtf8());
```

Comment:
seeing this pop up so often, I wonder why no make `boundary_` a QByteArray member and omit all this conversion business :)